examples/widgets/scale/lv_example_scale_10.c

```diff
@@ -0,0 +1,158 @@
+static int8_t hr_step = 1;
+static lv_obj_t * needle_line = NULL;
+static lv_obj_t * hr_value_label = NULL;
+static lv_obj_t * bpm_label = NULL;
+static lv_obj_t * scale = NULL;
+
+lv_color_t get_hr_zone_color(int32_t hr)
+{
+    if(hr < 117) return lv_palette_main(LV_PALETTE_GREY);      // Zone 1
+    else if(hr < 135) return lv_palette_main(LV_PALETTE_BLUE); // Zone 2
+    else if(hr < 158) return lv_palette_main(LV_PALETTE_GREEN);// Zone 3
+    else if(hr < 176) return lv_palette_main(LV_PALETTE_ORANGE);// Zone 4
+    else return lv_palette_main(LV_PALETTE_RED);               // Zone 5
+}
+
+static void hr_anim_timer_cb(lv_timer_t * timer)
+{
+    hr_value += hr_step;
+
+    if(hr_value >= 195) {
+        hr_value = 195;
+        hr_step = -1;
+    }
+    else if(hr_value <= 98) {
+        hr_value = 98;
+        hr_step = 1;
+    }
+
+    /* Update needle */
+    lv_scale_set_line_needle_value(scale, needle_line, -8, hr_value);
+
+    /* Update HR text */
+    lv_label_set_text_fmt(hr_value_label, "%d", hr_value);
+
+    /* Update text color based on zone */
+    lv_color_t zone_color = get_hr_zone_color(hr_value);
+    lv_obj_set_style_text_color(hr_value_label, zone_color, 0);
+    lv_obj_set_style_text_color(bpm_label, zone_color, 0);
+}
+
+void lv_example_scale_10(void)
+{
+    scale = lv_scale_create(lv_screen_active());
+    lv_obj_center(scale);
+    lv_obj_set_size(scale, 200, 200);
+
+    lv_scale_set_mode(scale, LV_SCALE_MODE_ROUND_INNER);
+    lv_scale_set_range(scale, 98, 195);
+    lv_scale_set_total_tick_count(scale, 15);
+    lv_scale_set_major_tick_every(scale, 3);
+    lv_scale_set_angle_range(scale, 280);
+    lv_scale_set_rotation(scale, 130);
+    lv_scale_set_label_show(scale, false);
+
+    lv_obj_set_style_length(scale, 6, LV_PART_ITEMS);
+    lv_obj_set_style_length(scale, 10, LV_PART_INDICATOR);
+    lv_obj_set_style_arc_width(scale, 0, LV_PART_MAIN);
+
+    // Helper macro to create section styles
+#define INIT_SECTION_STYLE(name, color)                                  \
+    static lv_style_t name##_items, name##_main, name##_indicator;       \
+    /*Minor ticks style*/                                                \
+    lv_style_init(&name##_items);                                        \
+    lv_style_set_line_color(&name##_items, color);                       \
+    lv_style_set_line_width(&name##_items, 0); \
+    /*Major ticks style*/                                                \
+    lv_style_init(&name##_indicator);                                    \
+    lv_style_set_line_color(&name##_indicator, color);                   \
+    lv_style_set_line_width(&name##_indicator, 0); \
+    /*Scale arc style*/                                                  \
+    lv_style_init(&name##_main);                                         \
+    lv_style_set_arc_color(&name##_main, color);                         \
+    lv_style_set_arc_width(&name##_main, 20);
+
+    // Helper macro to create and add section
```

Comment:
Please use this comment format
```suggestion
    /* Helper macro to create and add section */
```